docs/contributing/Compiler Test Plan.md

```diff
@@ -72,33 +74,37 @@ This document provides guidance for thinking about language interactions and tes
     - Ref / out parameters
     - Compound operators (+=, /=, etc ..) 
     - Assignment exprs
+- Ref returns
 
- 
 # Misc
 - reserved keywords (sometimes contextual)
 - pre-processing directives
 - COM interop
 - modopt and modreq
 - ref assemblies
 
-
 # Testing in interaction with other components
 Interaction with IDE, Debugger, and EnC should be worked out with relevant teams. A few highlights:
 - IDE
```

Comment:
The IDE test plan is at https://github.com/dotnet/roslyn/blob/master/docs/contributing/IDE%20Test%20Plan.md . Perhaps it would be better to refer to that?